src/ducks/categorization/services-fixtures.spec.js

```diff
@@ -426,7 +418,7 @@ xOrDescribe('Chain of predictions', () => {
         // Format results
         const fmtedResults = fmtResults(results)
         // Format results for the historized CSV
-        const fixtureCSV = fmtResultsCSV(results)
+        const fixtureCSV = fmtResultsCSV(results, bank)
```

Comment:
It's written bank here but in the signature of fmtResultsCSV, it's written cozyId  ?

Reply:
You're right. Actually the `banks/bank` naming had no meaning at all in this context. I replaced it with `cozyInstances/cozyId`